src/components/app.less

```diff
@@ -5,8 +5,8 @@
 
 	font-family: 'Helvetica Neue', arial, sans-serif;
 	font-weight: 400;
-	font-size: 16px;
-	line-height: 1.5;
+	font-size: 11px;
```

Comment:
Jeśli dodasz te style tutaj to zmniejszysz tekst na całej planszy - włącznie z buttonami itp. Chyba tego nie chcemy